commercetools-sdk-base/src/main/java/io/sphere/sdk/search/model/TermFilterExistsAndMissingSearchModel.java

```diff
@@ -0,0 +1,44 @@
+package io.sphere.sdk.search.model;
+
+import io.sphere.sdk.search.FilterExpression;
+
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ * Model to build term filters with exits and missing predicates.
+ * .
+ * @param <T> type of the resource
+ * @param <V> type of the value
+ */
+public interface TermFilterExistsAndMissingSearchModel<T, V> extends TermFilterSearchModel<T, V>, ExistsAndMissingFilterSearchModelSupport<T> {
+    @Override
+    List<FilterExpression<T>> containsAll(final Iterable<V> values);
+
+    @Override
+    List<FilterExpression<T>> containsAllAsString(final Iterable<String> values);
+
+    @Override
+    List<FilterExpression<T>> containsAny(final Iterable<V> values);
+
+    @Override
+    List<FilterExpression<T>> containsAnyAsString(final Iterable<String> values);
+
+    @Override
+    List<FilterExpression<T>> is(final V value);
+
+    @Override
+    List<FilterExpression<T>> isIn(final Iterable<V> values);
```

Comment:
As said in person, there are two reasons why you would like to override methods in an interface or abstract class:
1) to show the most important methods in bold in the IDE
2) to rename an argument, in case you can provide a better name in this context